src/openforms/scss/vendor/_django-two-factor-auth.scss

```diff
@@ -0,0 +1,6 @@
+.centered-image {
```

Comment:
oof this is a very generic classname, let's scope it to avoid unintended bleeding into other pages

```suggestion
body.login .centered-image {
```